lib/xterm.plugin.bash.sh

```diff
@@ -0,0 +1,32 @@
+# script taken straight from bash-it with almost no adaptation
+# just commented 3 lines and added bash-preexec.sh into .bashrc
+
+#cite about-plugin
+#about-plugin 'automatically set your xterm title with host and location info'
+
+_short-dirname () {
+  local dir_name=`dirs +0`
+  [ "$SHORT_TERM_LINE" = true ] && [ ${#dir_name} -gt 8 ] && echo ${dir_name##*/} || echo $dir_name
+}
+
+_short-command () {
+  local input_command="$@"
+  [ "$SHORT_TERM_LINE" = true ] && [ ${#input_command} -gt 8 ] && echo ${input_command%% *} || echo $input_command
+}
+
```

Comment:
i emphasize a bit with this, but i don't care enough for indentation nowadays...

coding style is exactly like css, and everyone will have their own.

but whatever, if it makes you happy, and since i'm here, set it all to 4 spaces in this file.